packages/@uppy/transloadit/src/index.js

```diff
@@ -37,6 +33,11 @@ const TL_COMPANION = /https?:\/\/api2(?:-\w+)?\.transloadit\.com\/companion/
 export default class Transloadit extends BasePlugin {
   static VERSION = packageJson.version
 
+  static COMPANION = 'https://api2.transloadit.com/companion'
+
+  // Regex matching acceptable postMessage() origins for authentication feedback from companion.
+  static ALLOWED_COMPANION_PATTERN = /\.transloadit\.com$/
+
```

Comment:
I though we would not do that, and instead define them as static only in the bundle?

Reply:
:thinking: